Dockerfile

```diff
@@ -18,20 +20,9 @@ RUN mkdir -p /opt/rcon-cli && \
     ln -s /opt/rcon-cli/rcon-cli /usr/bin && \
     rm /tmp/rcon-cli.tgz
 
-ENTRYPOINT ["/opt/backup-loop.sh"]
+CMD ["/opt/backup-loop.sh"]

```

Comment:
For me there's a blurry line between ENTRYPOINT and CMD. So I'm fine with reverting this. Why I've made this change:
```
# How to run bash / rcon-cli / ... when ENTRYPOINT is used
$ docker run --rm -it --entrypoint="bash" docker-mc-backup:v1
# How to run bash / rcon-cli / ... when CMD is used
$ docker run --rm -it docker-mc-backup:v2 bash
```
Since `backup-loop.sh` doesn't accept any arguments, for me it doesn't make sense to use it as `ENTRYPOINT`. But that's just a personal opinion.

I can add `ENTRYPOINT ["/bin/bash", "-c"]` to define some entrypoint explicitely.

Reply:
Agreed, this one especially is a blurry line since it doesn't take arguments, as you said; however, my general rule of thumb is that if the image generally should `run` without any extra args and do something when run, then `ENTRYPOINT` makes more sense. It's awkward to add `--entrypoint bash` to do arbitrary things with the image, but the explicit override makes sense to me for those one-off cases.

Reply:
So you'd prefer for me to revert this change - do I understand right?

Reply:
Reverted.